Libraries/LibWeb/DOM/EventListener.h

```diff
@@ -45,8 +45,28 @@ class EventListener
 
     JS::Function& function();
 
+    const StringView& type() const { return m_type; }
+    void set_type(const StringView& type) { m_type = type; }
+
+    bool capture() const { return m_capture; }
+    void set_capture(bool capture) { m_capture = capture; }
+
+    bool passive() const { return m_passive; }
+    void set_passive(bool passive) { m_capture = passive; }
+
+    bool once() const { return m_once; }
+    void set_once(bool once) { m_once = once; }
+
+    bool removed() const { return m_removed; }
+    void set_removed(bool removed) { m_removed = removed; }
+
 private:
+    StringView m_type;
```

Comment:
`StringView` should (almost) never be used as a member variable type, since there is no guarantee that the string data pointed to will remain valid. Instead, use a `FlyString` like we use for the event names elsewhere.